integration-tests/micrometer/src/main/java/org/apache/camel/quarkus/component/micrometer/it/MicrometerProducers.java

```diff
@@ -64,4 +67,16 @@ public Meter.Id map(Meter.Id id) {
         };
     }
 
+    @Singleton
+    @Produces
```

Comment:
Nitpick - That's the JAX-RS `@Produces`, not the CDI one.

Reply:
Thanks! fixed